docs/source/mturk.rst

```diff
@@ -0,0 +1,181 @@
+..
+  Copyright (c) 2017-present, Facebook, Inc.
+  All rights reserved.
+  This source code is licensed under the BSD-style license found in the
+  LICENSE file in the root directory of this source tree. An additional grant
+  of patent rights can be found in the PATENTS file in the same directory.
+
+Using Mechanical Turk
+===============
+
+In ParlAI, you can use Amazon Mechanical Turk for **data collection**, **training** and **evaluation** of your dialog model. 
+
+Human Turkers are viewed as just another type of agent in ParlAI, and hence person-to-person, person-to-bot, or multiple people and bots in group chat can all talk to each other within the same framework. 
+
+The human Turkers and other agents all communicate in observation/action dict format. During the conversation, the message that human Turkers receive is rendered on the live chat webpage in a pretty printed format, similar to the following:
+
+.. figure:: _static/img/mturk-small.png
+   :align: center
+
+   Example: Human Turker participating in a QA data collection task
+
+General Concepts
+---------------
+
+.. figure:: _static/img/mturk-flowchart.png
+   :width: 400px
+   :align: center
+
+   Diagram for a simple person-to-bot setup *
+
+Each MTurk task has at least one human Turker that connects to ParlAI via the Mechanical Turk Live Chat interface. 
+
+Each MTurk task must also have a local agent that runs on the ParlAI user's machine and drives the conversation with the Turker. In addition, the local agent is responsible for the following:
+
+1. Pulling data from datasets, and sending them as conversation context to the Turker.
+2. Feeding Turker's response into local dialog models, and sending model output back to Turker.
+3. Logging any part of the conversation.
+
+The logic of the local agent is implemented in its ``observe()`` and ``act()`` method.
+
+``observe()``
+^^^^^^
+
+When the Turker sends a response, the ``observe()`` method is called. We can get the Turker's response from ``self.observation``, which is populated automatically.
```

Comment:
probably good to change `observe()` to `observe(observation)`.

Then this text could say something like...
When the Turker sends a response, the ``observe`` method is called. The observation dict sent to this function contains all the information from the Turker, with the text the turker sent in the 'text' field.